tests/positive/general/preemption.go

```diff
@@ -37,7 +37,7 @@ func init() {
 // Ignition should apply.
 func makePreemptTest(components string) types.Test {
 	longnames := map[string]string{
-		"b": "base",
+		"b": "99-test-preemption",
```

Comment:
Why is this change needed?

Reply:
There's no specific reason as such but thought of providing a number prefix to show the higher priority. I think here, it would be better to keep this as-is. Let me revert this change.